sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CAutoConfiguration.java

```diff
@@ -89,9 +88,10 @@ private void sendTelemetry() {
         if (properties.isAllowTelemetry()) {
             final Map<String, String> events = new HashMap<>();
             final TelemetrySender sender = new TelemetrySender();
-
+            String tenantName = properties.getTenantName();
+            Assert.hasText(tenantName, "tenant name should contains text.");
```

Comment:
no asserts here, otherwise it will stop the application from starting.

Reply:
If we are going to check the validity of the properties, we should add it to the property class.